app/src/main/java/com/example/plantsapp/adapter/PlantComparator.kt

```diff
@@ -0,0 +1,15 @@
+package com.example.plantsapp.adapter
+
+import androidx.recyclerview.widget.DiffUtil
+import com.example.plantsapp.model.Plant
+
+class PlantComparator : DiffUtil.ItemCallback<Plant>() {
```

Comment:
1. It Is not a Comparator - it is ItemCallback, so PlantItemCallback would be more correct name